code/main/task/eventProcessor.brs

```diff
@@ -116,20 +97,11 @@ function _adb_EventProcessor(task as object) as object
             end if
 
             m._mediaModule.processEvent(requestId, data)
-        end function,
+            m.processQueuedRequests()
```

Comment:
This function is called in multiple places. Can we only call it after processing any events ( after this line ->  `m._processAdbRequestEvent(event)`)? This change should not affect performance, but it improves code clarity. What do you think?

Reply:
Yes, I initially kept it there, but I felt like we should call it from places it should be triggered for logic flow.

Reply:
Let's decide later after adding the Consent module.

Reply:
I too prefer calling it only after m._processAdbRequestEvent(event). The process function is going to exit early if there are no requests.

Reply:
Okay I will update to call it after adbrequestEvent

Reply:
@praveek @yangyansong-adbe 
I was actually changing that and it messes with the functional tests as it runs after every api call. Couple of test cases are failing and might have to do a bigger change to test that kind of workflow. As opposed to keeping where it is needed helps with ordering and unexpected request processing. We also have task which will be calling processQueuedRequests every 250ms in case the requests are queued due to missing config or network issue.